spec.html

```diff
@@ -31728,6 +31744,21 @@ <h1>Properties of the Array Prototype Object</h1>
         <p>The Array prototype object is specified to be an Array exotic object to ensure compatibility with ECMAScript code that was created prior to the ECMAScript 2015 specification.</p>
       </emu-note>
```

Comment:
Is this busted? There's no `</emu-clause>` on this and the indentation is off.

Reply:
GitHub can’t really handle expanding diffs on the web on a file as large as the spec.